netplugin/agent/agent.go

```diff
@@ -61,7 +61,8 @@ func NewAgent(pluginConfig *plugin.Config) *Agent {
 	// Initialize appropriate plugin
 	switch opts.PluginMode {
 	case "docker":
-		dockplugin.InitDockPlugin(netPlugin)
+	case "docker-compose":
```

Comment:
was trying to convey that it is docker and enables docker compose. Could change it to swarm-mode, if you don't have any other suggestion. let me know